modules/dapps.js

```diff
@@ -1069,7 +1069,13 @@ __private.createSandbox = function (dapp, params, cb) {
 				return setImmediate(cb, err);
 			}
 
-			var sandbox = new Sandbox(path.join(dappPath, 'index.js'), dapp.transactionId, params, __private.apiHandler, true);
+			var withDebug = false;
+			process.execArgv.forEach( function(item, index) {
+				if (item.indexOf('--debug') >= 0)
```

Comment:
I wouldn't recommend ~ because it's opaque, make the code less clear as most people don't know what it does. https://www.joezimjs.com/javascript/great-mystery-of-the-tilde/